.github/workflows/tests.yml

```diff
@@ -31,8 +31,6 @@ jobs:
         export PATH=/usr/bin:/bin:/usr/local/bin
         ruby uninstall -d >/dev/null
         ruby uninstall -f >/dev/null
-        ruby install --force-curl
```

Comment:
Let's keep this argument.

Reply:
Fixed by d6289ea9b573dda3c2804fe6dad6c336c92b7925

Reply:
Where was this even used before? I don't see it in the Ruby script.